InWorldz/InWorldz.ApplicationPlugins/Guest/GuestModule.cs

```diff
@@ -122,7 +122,7 @@ public void AddRegion(Scene scene)
         void EventManager_OnChatFromClient(object sender, OSChatMessage chat)
         {
             if (!m_enabled) return;
-            if (chat.Message == "" || chat.SenderUUID == chat.DestinationUUID) return;
+            if (String.IsNullOrEmpty(chat.Message) || chat.SenderUUID == chat.DestinationUUID) return;
```

Comment:
Semantic change: null chat.Message now causes early return.  This means that the "Guest" lastname message filter is not run.  That makes sense: don't try string replaces on a null string!